spec/controllers/link_check_reports_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "spec_helper"
+require "services"
+
+describe LinkCheckReportsController, type: :controller do
+  describe "#create" do
+    let(:user) { create(:user) }
+    let(:manual) { FactoryGirl.build(:manual, id: 1, body: "[link](http://www.example.com)") }
+    let(:section) { FactoryGirl.create(:section_edition, id: 1) }
+
+    let(:link_checker_api_response) do
+      {
+        batch_id: 1,
+        completed_at: nil,
+        status: "in_progress",
+        links: [
+          {
+            uri: "http://www.example.com",
+            status: "error",
+            checked: Time.parse("2017-12-01"),
+            warnings: ["example check warnings"],
+            errors: ["example check errors"],
+            problem_summary: "example problem",
+            suggested_fix: "example fix"
+          }
+        ]
+      }
+    end
+
+    before do
+      login_as_stub_user
+      allow(Services.link_checker_api).to receive(:create_batch).and_return(link_checker_api_response)
+      allow(Manual).to receive(:find).and_return(manual)
```

Comment:
Could we use `FactoryGirl.create` so we have a real object and avoid the need to stub the `find`?

Reply:
Here, and in a few other places.